array.js

```diff
@@ -0,0 +1,168 @@
+'use strict';
+
+var i = 600;
+
+function genHourlyReport(location) {
+  for (i = 600; i <= 2000; i += 100) {
+    var randGen = Math.floor(Math.random() * (location.maxCust - location.minCust + 1)) + location.minCust;
+    if (i < 1000) {
+      location.genResults.push('<li> 0' + i + ' hours:' + randGen + ' cookies </li>');
+    } else {
+      location.genResults.push('<li>' + i + ' hours:' + randGen + ' cookies</li>');
+    }
+  }
+
+  console.log(location.genResults);
+}
+
+var pike = {
+  name: 'pike',
+  minCust: 23,
+  maxCust: 65,
+  avgCust: 6.3,
+  genResults: [],
+  gen: function () {
+    genHourlyReport(pike);
+  },
+};
+
+var seaTac = {
+  name: 'seaTac',
+  minCust: 3,
+  maxCust: 24,
+  avgCust: 1.2,
+  genResults: [],
+  gen: function () {
+    return genHourlyReport(seaTac);
+  },
+
+};
+
+var seaCenter = {
+  name: 'seaCenter',
+  minCust: 11,
+  maxCust: 38,
+  avgCust: 3.7,
+  genResults: [],
+  gen: function () {
+    return genHourlyReport(seaCenter);
+  },
+};
+
+var theHill = {
+  name: 'theHill',
+  minCust: 20,
+  maxCust: 38,
+  avgCust: 2.3,
+  genResults: [],
+  gen: function () {
+    return genHourlyReport(theHill);
+  },
+};
+
+var alki = {
+  name: 'alki',
+  minCust: 2,
+  maxCust: 16,
+  avgCust: 4.6,
+  genResults: [],
+  gen: function () {
+    return genHourlyReport(alki);
+  },
+};
+
+pike.gen();
+seaTac.gen();
+seaCenter.gen();
+theHill.gen();
+alki.gen();
+
+var locationObjects = [];
+locationObjects.push(pike.name);
+locationObjects.push(seaTac.name);
+locationObjects.push(seaCenter.name);
+locationObjects.push(theHill.name);
+locationObjects.push(alki.name);
+console.log(locationObjects);
+var reportResults = [];
+reportResults.push(pike.genResults);
+reportResults.push(seaTac.genResults);
+reportResults.push(seaCenter.genResults);
+reportResults.push(theHill.genResults);
+reportResults.push(alki.genResults);
+console.log(reportResults);
+
+var ul = document.createElement('ul');
+document.body.appendChild(ul);
+
+for (var i = 1; i <= reportResults.length; i++) {
+  var li = document.createElement('li');
+  li.className = 'report';
+  var a = document.createElement('a');
+  a.innerHTML = locationObjects[i] + reportResults[i];
+
+  li.appendChild(a);
+  ul.appendChild(li);
+}
+
+//populate array with randos
```

Comment:
try not to commit commented out code, if you want to be able to see previous or outdated code just make another commit.